experimental/dds/tree2/src/test/domains/json/benchmarks.ts

```diff
@@ -38,7 +37,7 @@ export function sumMap(cursor: ITreeCursor): number {
 	return total;
 }
 
-export function sumDirect(jsonObj: Jsonable): number {
+export function sumDirect(jsonObj: any): number {
```

Comment:
Could this be typed as a `Record` instead? If not, `unknown`?

Reply:
`Record` can't be used because it might be given any type.
`Object.values` doesn't handle `unknown`.
This function actually handles `any` and doesn't require the argument to be "Jsonable". The thing it doesn't handle (aka doesn't produce the result I would have expected) is a being given a scalar `number`. That will return 0. In all other cases it finds those numbers.
I originally tried to use a more sophisticated type, but `any` really fits.

Reply:
I think the implied assumption is that jsonObj is an 'object'.

Reply:
Name certainly implies that but that isn't how it is called, at least not by itself.